UnitTestBot/UTBotJava
# Fix problems with python execution

## Description

* no minimization when the tested module is also installed from pip (as #1945): order in `sys.path` have been changed
* removed module `coverage` and replaced it with a builtins module `trace`
* fixed problem with one execution timeout: previous one timeout`socket.soTimeout` had been used by some executions instead only one, current realization create a thread for execution and interrupt it after timeout
* update `utbot_executor`, new version `1.3.1`: more logs in `DEBUG` mode and fix problems with coverage

Fixes #1945

## How to test

### Manual tests

* minimization and coverage: see #1945 (with or without installed module `langchain`)
* timeout: expected that this function doesn't have any tests (in future we can generate tests with timeout same as UtBotJava, see #1939)
```python
import time
def f(x: int):
    time.sleep(10)
    return x
```

## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.
